readthedocs/gold/views.py

```diff
@@ -238,13 +236,14 @@ def post(self, request, format=None):
                     # Gold Membership
                     user = User.objects.get(username=username)
                     subscription = stripe.Subscription.retrieve(event.data.object.subscription)
-                    log.bind(stripe_plan=subscription.plan.id)
+                    stripe_price = subscription["items"].data[0].price
```

Comment:
I think it makes sense to log the cases where there are more prices/plans than the ones we are expecting. This will help us to track this down in case of an issue

```suggestion
                    stripe_price = subscription["items"].data[0].price
                    if len(subscription["items"].data) > 1:
                        log.info("This user/organization has more than one plan/price associated", .... extra keys ...)
```